src/app.ts

```diff
@@ -1,13 +1,39 @@
 import express from "express";
 import cors from "cors";
 import bodyParser from "body-parser";
+import { MongoClient, ServerApiVersion } from "mongodb";
+require("dotenv").config();
 
 const PORT = 3000;
 const app = express();
 import userRouter from "./routes/userRoutes";
-
+const dbUser = process.env.MONGO_DB_USER;
+const dbPassword = process.env.MONGO_DB_PASSWORD;
+const uri = `mongodb+srv://${dbUser}:${dbPassword}@secradardb.f9xsna0.mongodb.net/?retryWrites=true&w=majority`;
+console.log(uri);
```

Comment:
You accidentally left a console.log, delete it

Reply:
Removed it, thanks!